### ISSUE TITLE
JavaScript tracking doesn’t work on Safari (and maybe Chrome?) with X-Content-Type: nosniff

### ISSUE THREAD

**Issue by MatzeKitt** [pluginkollektiv/statify#58]
I generally enable `X-Content-Type: nosniff` on all of my websites because it’s an easy way to preserve Cross-Site-Scripting. Unfortunately, after enabling Cachify and thus also enabling the JavaScript tracking method in Statify Safari is unable to be tracked.

In the console of Safari the following error message appears:
```
Did not load script at 'https://kittpress.com/?statify_referrer=&statify_target=%2F' because non script MIME types are not allowed when 'X-Content-Type: nosniff' is given.
```

You can test it by yourself if you visit https://kittpress.com.

By using Chrome there isn’t even any entry in the network tab that refers to the Statify JavaScript caching and as far as I interpret my tracking statistics in the Dashboard there isn’t any tracking for Chrome users at all.

Maybe it could be enough to properly send a valid mime type?

**Comment by stklcode** [comment]
Confirmed at your and one of my sites.

In fact it's not only Safari, but at least Opera and Chrome show the same behavior, so probably all WebKit-based browsers.

**Comment by patrickrobrecht** [comment]
Fixed in #68
